delft3dworker/models.py

```diff
@@ -1140,14 +1163,6 @@ def update_task_result(self):
             self.task_uuid = None
             self.save()
 
-        # Pending means not queued yet or unknown
```

Comment:
REMOVED: If a tasks always runs longer than the heartbeat, we never get the result.